stress/shell/src/main/java/alluxio/stress/cli/MasterBatchTask.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.stress.cli;
+
+import alluxio.stress.Summary;
+import alluxio.stress.master.MasterBatchTaskParameters;
+import alluxio.util.JsonSerializable;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParametersDelegate;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Multiple tasks runner for Master StressBench.
+ */
+public class MasterBatchTask extends BatchTask {
+  private static final Logger LOG = LoggerFactory.getLogger(MasterBatchTask.class);
+  private static final int TARGETTHROUGHPUT = 1000000;
```

Comment:
```suggestion
  private static final int TARGET_THROUGHPUT = 1_000_000;
```

Reply:
why 1million?

Reply:
I just want a very big number, and I saw in the test plan zhenhua used it was 1 million. So I used this number.

Reply:
I see, for these heuristic numbers, if you have a reason why you put this number, pls leave a comment too.